### PR TITLE
Update FAQs question covering datasets with conflicting bounds

### LINKED ISSUE #453: [Doc]: Fix incorrect API reference in FAQ
### Describe your documentation update

In the FAQ for "How does `xcdat` interpret dataset metadata", `ds.temporal.average()` is incorrect referenced with an `axis` arg. `axis` is not an argument for this API.

We should reference `ds.bounds.add_bounds()` instead.

### LINKED ISSUE #473: [Doc]: Update FAQ for handling conflicting variable values
### Describe your documentation update

Related to #451, #470, and #143

* https://xcdat.readthedocs.io/en/latest/faqs.html#how-do-i-open-a-multi-file-dataset-with-values-that-conflict

Additional Items:
- [x] Check why old FAQ answer doesn't work anymore -- just forgot to add `coords="minimal"` this whole time
- [x] Check why xarray can open files without error  -- defaults to ``data_vars="all"` which concatenates dimensions
- [ ] Add `coords="minimal" and "join="left"` or "join="override"` to current option
- [ ] Add option for dropping conflicting variable (e.g., `drop_variables="lon_bnds"`)

### PR DESCRIPTION
**Pull request by tomvothecoder**
## Description

- Closes #453 
- Closes #473 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

If applicable:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass with my changes (locally and CI/CD build)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have noted that this is a breaking change for a major release (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: docs/faqs.rst [hunk 1]
```diff
@@ -156,24 +156,72 @@ A few workarounds include:
...
+to be combined. You can skip this check by specifying compat="override".``
 
 If you still intend on working with these datasets and recognize the source of the issue
-(e.g., minor floating point diffs), follow the instructions below.
-**Please understand the potential implications before proceeding!**
+(e.g., minor floating point diffs), follow the workarounds below. **Please proceed with
+caution. You should understand the potential implications of these workarounds.**
 
-.. code-block:: python
+1. Pick the first bounds variable and keep dimensions the same as the input dataset
```
Comment (tomvothecoder) [inline docs/faqs.rst:170]:
```suggestion
1. Pick the first bounds variable and keep dimensions the same as the input files
```

File: docs/faqs.rst [hunk 2]
```diff
@@ -156,24 +156,72 @@ A few workarounds include:
...
 
-    >>> xcdat.open_mfdataset("path/to/files/*.nc", compat="override", join="override")
+   - This option is recommended if you know bounds values should be the same across all
+     files, but one or more files has inconsistent bounds values which breaks the
+     concatenation of files.
```
Comment (tomvothecoder) [inline docs/faqs.rst:174]:
```suggestion
     concatenation of files into a single `xr.Dataset` object.
```

**Review by tomvothecoder — COMMENTED** [review]
Hi @lee1043 and @pochedls, this PR is ready for review.

File: docs/faqs.rst [hunk 3]
```diff
@@ -156,24 +156,72 @@ A few workarounds include:
...
+        dimension
+      - xarray defaults to ``join="outer"``. This can cause issues where data
+        variable values conflict because additional coordinates points are
+        concatenated at the point of conflict which can produce ``nan`` values.
+
+2. Drop the conflicting bounds variable(s)
+
+   - This option is recommended if you know don't mind dropping the bounds variable(s).
+     You can add new bounds using the ``add_bounds`` keyword argument (defaults to
+     ``["X", "Y"]``).
```
Comment (pochedls) [inline docs/faqs.rst:215]: Suggest: 

> xCDAT will generate and replace the dropped bounds if `add_bounds` includes the axis for the dropped variable (by default, `add_bounds=["X", "Y"]`).

**Review by pochedls — APPROVED** [review]
Looks great – thanks @tomvothecoder

**Pull request merged**